src/ontology/imports/foodon_product_import.owl

```diff
@@ -33790,6 +33790,61 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03305484"/>
+        <obo:IAO_0000115 xml:lang="en">A powder made from the leaves of kratom plant (Mitragyna speciosa).</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Mitragyna_speciosa</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-07T12:58:12Z</dc:date>
+        <rdfs:label xml:lang="en">kratom leaf powder</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004175 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004175">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00004174"/>
+        <obo:IAO_0000115 xml:lang="en">A powder made from a leaf of kratom plant (Mitragyna speciosa) which originates from Indonesia.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://kratomspot.com/indo-kratom/</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-07T13:03:21Z</dc:date>
+        <oboInOwl:hasNarrowSynonym xml:lang="en">indo kratom powder</oboInOwl:hasNarrowSynonym>
+        <rdfs:comment xml:lang="en">Indo kratom powder comes in different color varieties: white, green, and red. The color refers to the color of the kratom leaf veins and stems at the time of harvest.</rdfs:comment>
+        <rdfs:label xml:lang="en">indo kratom leaf powder</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004176 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004176">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03303380"/>
+        <obo:IAO_0000115 xml:lang="en">Dry brown powder made from the blend of ground spices.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Five-spice_powder</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-07T14:07:28Z</dc:date>
+        <oboInOwl:hasNarrowSynonym xml:lang="en">brown powder</oboInOwl:hasNarrowSynonym>
+        <rdfs:comment xml:lang="en">A common mix is: star anise, clove, chinese cinnamon, sichuan pepper, fennel seed.</rdfs:comment>
+        <rdfs:label xml:lang="en">chinese five spice powder</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004177 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004177">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002501"/>
+        <obo:IAO_0000115 xml:lang="en">A food product that contains more than one protein source.</obo:IAO_0000115>
```

Comment:
Do we know or want to qualify anything about the product format?  Is this a blended powder or liquid?  How do we avoid from an ontological perspective that this can match any multi-ingredient food?